src/components/AppLayout.jsx

```diff
@@ -57,48 +72,61 @@ const AppLayout = () => {
   }
 
   return (
-    <Layout>
-      <Sidebar>
-        <Nav>
-          <NavItem>
-            <NavLink {...makeProps(['timetable'])}>
-              <NavIcon icon={CalendarIcon} />
-              <NavText>{t('Sidebar.timetable')}</NavText>
-            </NavLink>
-          </NavItem>
-          <NavItem>
-            <NavLink {...makeProps(['homeworks'])}>
-              <NavIcon icon={CheckboxIcon} />
-              <NavText>{t('Sidebar.homeworks')}</NavText>
-            </NavLink>
-          </NavItem>
-          <NavItem>
-            <NavLink {...makeProps(['grades'])}>
-              <NavIcon icon={PieChartIcon} />
-              <NavText>{t('Sidebar.grades')}</NavText>
-            </NavLink>
-          </NavItem>
-          <NavItem>
-            <NavLink {...makeProps(['presence'])}>
-              <NavIcon icon={WalkIcon} />
-              <NavText>{t('Sidebar.presence')}</NavText>
-            </NavLink>
-          </NavItem>
-        </Nav>
-      </Sidebar>
-      <BarComponent />
-      <Main>
-        <Content>
-          {isMobile && (
-            <BarCenter>
-              <Typography variant="h5">{client.appMetadata.slug}</Typography>
-            </BarCenter>
+    <AccountContext.Provider value={{ currentAccount, setCurrentAccount }}>
```

Comment:
I recommend that you extract your provider into a separate file, such as SelectionProvider inside Drive ([code](https://github.com/cozy/cozy-drive/blob/master/src/modules/selection/SelectionProvider.jsx)) and after to use it inside `AppProvider` to make it more readable

Reply:
It has been added here -> https://github.com/cozy/cozy-papillon-app/pull/29/commits/d36b4d535e5a9c0b648836dfef4bde9d91d7f531